### PR TITLE
Fixing Issue with unexposed entry files

### PR DESCRIPTION
**Pull request by TheMcMurder**
Fixes https://github.com/esm-bundle/rxjs/issues/81

### CODE REVIEW

File: src/es5/rxjs-testing.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from "rxjs/_esm5/testing/index";
```
Comment (TheMcMurder) [inline src/es5/rxjs-testing.js:1]: I'm not really sure this is necessary to expose but I figured because I was here anyways I'd do it

**Review by joeldenning — APPROVED** [review]
Looks good to me. I think that this should be published manually, instead of through CI in Travis. Perhaps publish it as `6.6.3-fix.0`? The `-fix.0` suffix is one I used in the past when updating an already-published version

**Comment by TheMcMurder** [comment]
Yeah I can publish it manually. I'm not 100% sure on how to do that given the existing automation but this is how I think it would be done:

1. Merge
- Automation wouldn't republish because there is an existing version that matches
2. Checkout master an build the project
3. Run yarn publish on my local machine with the version being `6.6.3-fix.0`

Is that right?

**Comment by joeldenning** [comment]
Yes that is correct.

**Pull request merged**